go.mod

```diff
@@ -19,7 +19,6 @@ require (
 	k8s.io/apimachinery v0.17.1
 	k8s.io/client-go v0.17.1
 	k8s.io/klog v1.0.0
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.0
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.5
 )
 
-replace sigs.k8s.io/apiserver-network-proxy/konnectivity-client => ./konnectivity-client
```

Comment:
I think we should keep this "replace", following the [example](https://github.com/kubernetes/kubernetes/blob/f7eafa1a838c097c9b6fff118e24bf8334c34f32/go.mod#L537) of Kubernetes staging repositories. This makes sure the code in master branch works together.

We need to have a skew test between the client and the server, but that's a separate issue.

Reply:
I imagined a scenario where the proxy and client code have different versions, but in theory that should never happen since we synchronize the releases. Hmm, if the master branch of the two libraries are always compatible, I guess we have no need for this PR since any version of the proxy code contains the same version client code.

Reply:
If we check in more code into the proxy client but haven't yet cut a release, this PR will leave the code untested.